### PR TITLE
WT-7847 Send Inventory Data ruby mws

### PR DESCRIPTION
**Pull request by sachinmaharjan**
Story: https://jira.blurb.com/browse/WT-7847
Send Inventory data to Amazon MWS
Quantity for Amazon Books

### CODE REVIEW

File: spec/ruby-mws/api/feed_spec.rb [hunk 1]
```diff
@@ -370,6 +396,38 @@
...
+          info.feed_type.should == MWS::API::Feed::PRODUCT_LIST_INVENTORY
+        end
+
+        it "should create the correct body for inventory" do
+          MWS::API::Feed.should_receive(:post) do |uri, hash_list|
+            hash_list.should include(:body)
+            body = hash_list[:body]
+            body_doc = Nokogiri.parse(body)
+
+            body_doc.css('AmazonEnvelope Message Inventory SKU').should_not be_empty
```
Comment (nhullrichter-blurb) [inline spec/ruby-mws/api/feed_spec.rb]: This is tested below.

File: spec/ruby-mws/api/feed_spec.rb [hunk 2]
```diff
@@ -370,6 +396,38 @@
...
+            body_doc.css('AmazonEnvelope Message MessageID')[0].text.should == "1"
+            body_doc.css('AmazonEnvelope Message MessageID')[1].text.should == "2"
+            body_doc.css('AmazonEnvelope Message Inventory SKU')[0].text.should == "9781320717869"
+            body_doc.css('AmazonEnvelope Message Inventory SKU')[1].text.should == "9781320717870"
+            body_doc.css('AmazonEnvelope MessageType').length.should == 1
+            body_doc.css('AmazonEnvelope PurgeAndReplace').text.should == "false"
+            body_doc.css('AmazonEnvelope Message Inventory').should_not be_empty
```
Comment (nhullrichter-blurb) [inline spec/ruby-mws/api/feed_spec.rb]: ditto

**Comment by nhullrichter-blurb** [comment]
+1 after comments.

File: spec/ruby-mws/api/feed_spec.rb (hunk 1 above)
Comment (sachinmaharjan) [inline spec/ruby-mws/api/feed_spec.rb]: Removed

File: spec/ruby-mws/api/feed_spec.rb (hunk 2 above)
Comment (sachinmaharjan) [inline spec/ruby-mws/api/feed_spec.rb]: Removed

**Comment by sachinmaharjan** [comment]
@pierrel When u get a chance.

**Comment by pierrel** [comment]
+1

**Pull request merged**